backend/api.test/Database/DatabaseUtilities.cs

```diff
@@ -93,31 +101,50 @@ public async Task<MissionRun> NewMissionRun(
             return missionRun;
         }
 
-        public async Task<Installation> NewInstallation()
+        public async Task<Installation> ReadOrNewInstallation()
         {
+            if (await _installationService.ReadByInstallationCode(_testInstallationCode) is Installation installation) return installation;
```

Comment:
Is it safe for the tests to potentially share the same installation/plant/deck/area objects? Should be fine here since it is readOnly, but can theoretically still cause conflicts when writing and then reading

Reply:
Not 100% sure, the readOnly + the fact that all tests are marked with
`[Collection("Database collection")]` should make sure that we dont get any conflicts, but tbh Im not completely sure.